cumulusci/core/config/OrgConfig.py

```diff
@@ -268,10 +268,13 @@ def installed_packages(self):
             _installed_packages = defaultdict(list)
             for isp in isp_result["records"]:
                 sp = isp["SubscriberPackage"]
-                spv_result = self.salesforce_client.restful(
-                    "tooling/query/?q=SELECT Id, MajorVersion, MinorVersion, PatchVersion, BuildNumber, "
-                    f"IsBeta FROM SubscriberPackageVersion WHERE Id='{isp['SubscriberPackageVersionId']}'"
-                )
+                try:
+                    spv_result = self.salesforce_client.restful(
+                        "tooling/query/?q=SELECT Id, MajorVersion, MinorVersion, PatchVersion, BuildNumber, "
+                        f"IsBeta FROM SubscriberPackageVersion WHERE Id='{isp['SubscriberPackageVersionId']}'"
+                    )
+                except SalesforceError:
+                    continue
```

Comment:
Shall we add logging here? I'm afraid we'll end up with confusing preflight failures in the future if we don't somehow record that we failed to find details of some package or other.

Reply:
Good call. Added.